contents/handbook/small-teams/monitoring/index.mdx

```diff
@@ -14,12 +14,10 @@ import Objectives from "./objectives.mdx"
 
 <Mission />
 
-## Objectives
+## Q4 2023 Goals
 
 <Objectives />
 
-Follow along in [our Q3 MegaIssue](https://github.com/PostHog/posthog/issues/16287)
```

Comment:
do we want a Q4 mega issue?

Reply:
Yeah, I think it would be good to have one, so that we can list a couple of ideas for you to work on Paul 😄